### ISSUE TITLE
split-platform-components and Platform module / Expo

### ISSUE THREAD

**Issue by Palisand** [Intellicode/eslint-plugin-react-native#163]
Seeing as how many eslint question seem to be ignored on StackOverflow, I've decided to ask this here.

Should the [split-plaform-components rule](https://github.com/Intellicode/eslint-plugin-react-native/blob/master/docs/rules/split-platform-components.md) be ignored when developing an RN app with Expo? In a more general sense, is there a subset of rules that shouldn't apply to apps being developed with this toolset?

Perhaps a better question is: **Should this rule not raise a warning when the file in question is using the Platform module?**

```javascript
Platform.select({
  ios: <IOSonlyComponent/>,
  android: <AndroidOnlyComponent/>
})
```

**Comment by Intellicode** [comment]
In your particular example, you probably should put your platform dependend component into the *.ios.js and *.android.js pattern, and use it like this
```import SomeComponent from './component',```
For which the React Native bundler will automatically resolve the correct component per platform.
